lib/rbtrace/cli.rb

```diff
@@ -407,6 +415,10 @@ def self.run
           tracer.puts "=> #{res}"
         end
 
+      elsif opts[:interactive_given]
+        require "rbtrace/interactive/#{opts[:interactive]}"
+        load `which #{opts[:interactive]}`.chomp
```

Comment:
Can we do this without shelling out to which? File.expand_path(,__FILE__) should work